README.md

```diff
@@ -20,12 +20,29 @@ Two Slurm scripts are included in this repo.  If you want to run Rstudio to do d
 
 To run the pipeline as a non-interactive batch job, open `run_scmake.slurm` and modify parameters like job length (`--time`) and partition (`-p`) as needed for how long you expect the job to run (see [this issue](https://github.com/USGS-R/national-flow-observations/issues/4) for estimates). Submit the job with `sbatch --mail-user=$USER@usgs.gov run_scmake.slurm`.
```

Comment:
It would be worth mentioning that people have to install their own `saml2aws`. You could link to the instructions [here](https://dsp-manual.wma.chs.usgs.gov/docs/aws/introduction_to_aws/).